src/SwiftReachApi/SwiftReachApi.php

```diff
@@ -32,26 +32,55 @@ class SwiftReachApi
      */
     private $base_url;
 
+    /**
+     * @var array
+     * default request options
+     */
+    private $defaults;
```

Comment:
I think a more descriptive name than' defaults', would be best.  Like your comment maybe 'request_options'.